testsuite/integration/basic/pom.xml

```diff
@@ -380,25 +529,41 @@
         </dependency>
         <dependency>
             <groupId>${project.groupId}</groupId>
-            <artifactId>jipijapa-spi</artifactId>
+            <artifactId>jipijapa-spi-jakarta</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>wildfly-clustering-common</artifactId>
```

Comment:
This dependency is unnecessary.

Reply:
@pferraro I will try manually removing it